### ISSUE TITLE
Elliptic Curve Digital Signature Algorithm

### ISSUE THREAD

**Issue by jturner** [somesocks/lua-lockbox#2]
Any plans on adding an Elliptic Curve Digital Signature Algorithm such as Secp256k1? Or any other public key algorithms?

**Comment by somesocks** [comment]
Yep, there sure are!  Public-key cryptosystems come in right after CSPRNGs and big-integer arithmetic.  If you have any more specific requests, open a separate issue, and I'll put them in the roadmap.
